### PR TITLE
Price option names are now human-friendly 

### LINKED ISSUE #16: Human-friendly price option names.
The filtered price options are currently named after the corresponding session length as a number of seconds.

This needs to be changed to use human-friendly duration strings, in the event that they are shown in any EDD interface.

Carbon has a sane way of achieving this that also supports L10n. See: [`CarbonInterval`](https://carbon.nesbot.com/docs/#api-interval).

### PR DESCRIPTION
**Pull request by mecha**
Resolves #16

### CODE REVIEW

File: src/Module/GetEddServicePricesCapableTrait.php [hunk 1]
```diff
@@ -55,10 +56,14 @@ protected function _getEddServicePrices($serviceId)
         $index   = 0;
 
         foreach ($lengths as $_lengthInfo) {
+            $_length = (int) $this->_containerGet($_lengthInfo, 'sessionLength');
+            $_name   = CarbonInterval::seconds($_length)->cascade()->forHumans();
```
Comment (XedinUnknown) [inline src/Module/GetEddServicePricesCapableTrait.php:60]: I wonder from where Carbon knows how to localize the human-readable length.

Comment (mecha) [reply src/Module/GetEddServicePricesCapableTrait.php:60]: It uses the PHP locale, according to its [documentation](https://carbon.nesbot.com/docs/#api-localization).

Comment (XedinUnknown) [reply src/Module/GetEddServicePricesCapableTrait.php:60]: I must say that I dislike the static part very, very much.

Comment (mecha) [reply src/Module/GetEddServicePricesCapableTrait.php:60]: Same. But it beats the alternative right now; implementing our own 😆 

FYI: I looked at other date time packages, such as Brick's. But they all seem to enjoy using static calling. Crazy!

**Edit**: A few links would be handy, no?

* https://github.com/brick/date-time
* https://github.com/danielstjules/php-pretty-datetime

Comment (XedinUnknown) [reply src/Module/GetEddServicePricesCapableTrait.php:60]: @mecha, Carbon docs appear to indicate that it's possible to work with instances too. But if I understood correctly, they are instances of actual datetimes, not of the library which has the functionality to deal with them.

**Review by XedinUnknown — APPROVED** [review]

**Pull request merged**